docs/en/sql-reference/aggregate-functions/parametric-functions.md

```diff
@@ -487,24 +487,23 @@ Where:
 
 ## uniqUpTo(N)(x)
 
-Calculates the number of different argument values ​​if it is less than or equal to N. If the number of different argument values is greater than N, it returns N + 1.
+Calculates the approximate number of different values of the argument up to a specified limit, `N`. This function is useful when you want to get a rough estimate of the number of unique values in a column but you're not interested in the exact count due to performance considerations. If the number of different argument values is greater than `N`, this function returns `N` + 1.
```

Comment:
> rough estimate

It gives a precise value, not a rough estimate.

Reply:
> You can use `uniqUpTo` to generate a report that shows only the keywords that produced at least 5 unique users.

Yes, so the use-case is very clear, and all the calculations are precise.

Reply:
Gotcha. How's this:

```suggestion
Calculates the number of different values of the argument up to a specified limit, `N`. If the number of different argument values is greater than `N`, this function returns `N` + 1, otherwise it calculates the exact value. 
```